docs/src/prototypes/chatPane/chatPaneContent.tsx

```diff
@@ -46,6 +68,17 @@ class ChatPaneContainer extends React.PureComponent<ChatPaneContainerProps> {
       scrollRef.scrollToBottom()
     }
   }
+
+  private getMessagePreviewForScreenReader(props) {
+    // Show the first 60 characters from the message.
+    // If to show more, NVDA splits it into 2 lines
+    const messageText = props.text || ''
+    if (messageText.length > 60) {
+      return `${messageText.slice(0, 60)} ..., by ${props.author}`
```

Comment:
as I see, two identical strings (in a sense of composing logic) are returned. In that case this could be simplified to the following code, no branching is necessary

```js
const messageText = props.text || ''
return `${messageText.slice(0, 60)} ..., by ${props.author}`
```